pkg/types/v1/luet.go

```diff
@@ -35,20 +43,70 @@ type Luet struct {
 	VerifyImageUnpack bool
 }
 
-func NewLuet(log Logger, context *context.Context, auth *dockTypes.AuthConfig, plugins ...string) *Luet {
-	bus.Manager.Initialize(context, plugins...)
-	if len(bus.Manager.Plugins) != 0 {
-		log.Infof("Enabled plugins:")
-		for _, p := range bus.Manager.Plugins {
-			log.Infof("* %s (at %s)", p.Name, p.Executable)
+type LuetOptions func(l *Luet) error
+
+func WithLuetPlugins(plugins []string) func(r *Luet) error {
```

Comment:
small style nit, if you make this
```suggestion
func WithLuetPlugins(plugins ...string) func(r *Luet) error {
```
makes it easier to the caller, but that's just about taste and consistency..

Reply:
mmh, thats a good point, but the only time we use it we already got a []string .... but indeed, for manual calling it should be better that way!